shared/wallets/send-form/container.js

```diff
@@ -1,11 +1,16 @@
 // @flow
 import SendForm from '.'
+import * as WalletsGen from '../../actions/wallets-gen'
 import {connect, type TypedState} from '../../util/container'
 
 const mapStateToProps = (state: TypedState) => ({})
 
 const mapDispatchToProps = (dispatch, {navigateUp}) => ({
-  onClose: () => dispatch(navigateUp()),
+  onClose: () => {
+    dispatch(navigateUp())
+    dispatch(WalletsGen.createClearBuildingPayment())
```

Comment:
maybe better to handle this stuff as a saga. if you look at signup/login/ etc they watch for nav up and cancel things